### PR TITLE
Add the acting user to the events in the audit log.

### PR DESCRIPTION
**Pull request by C0rby**
Added the acting user to the events in the audit log.

### CODE REVIEW

**Comment by C0rby** [comment]
I need to add checks for the executant since it can be nil.

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiSpaces-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/12653/38/1) failed. Further test are cancelled...

File: extensions/audit/pkg/types/conversion.go [hunk 1]
```diff
@@ -234,8 +234,8 @@ func FilesAuditEvent(base AuditEvent, itemid, owner, path string) AuditEventFile
 
 // ContainerCreated converts a ContainerCreated event to an AuditEventContainerCreated
 func ContainerCreated(ev events.ContainerCreated) AuditEventContainerCreated {
-	iid, path, uid := extractFileDetails(ev.Ref, ev.Executant)
-	base := BasicAuditEvent(uid, "", MessageContainerCreated(iid), ActionContainerCreated)
+	iid, path, uid := extractFileDetails(ev.Ref, ev.Owner)
+	base := BasicAuditEvent(uid, "", MessageContainerCreated(ev.Executant.OpaqueId, iid), ActionContainerCreated)
```
Comment (kobergj) [inline extensions/audit/pkg/types/conversion.go:238]: Not critical. But better to use `ev.Executant.GetOpaqueId()` to avoid panic when executant is nil

**Review by kobergj — APPROVED** [review]
One Suggestion, but not critical

File: extensions/audit/pkg/types/conversion.go (hunk 1 above)
Comment (C0rby) [reply extensions/audit/pkg/types/conversion.go:238]: True, I'll change that. :+1:

**Review by kobergj — APPROVED** [review]
Perf now :+1:

**Pull request merged**